### PR TITLE
[ref] Migrate CallToActionSection component

### LINKED ISSUE #197: [ref] Migrate `CallToActionSection` component

### PR DESCRIPTION
**Pull request by Joel-leal**
# Description

This PR brings significant improvements by simplifying the code structure, reducing reliance on external libraries, and increasing design flexibility. Specifically, the Call to Action section has been refactored to use utility classes from TailwindCSS instead of Chakra UI components, resulting in cleaner and more maintainable code. Additionally, the Illustration component now directly renders SVGs, eliminating the need for the Chakra UI Icon component, further streamlining the implementation.

## Changes

- ref: Adjusted the HTML structure and styling of the section.
- ref: Replaced the use of <Heading> and <Stack> components with a more concise and simplified approach using TailwindCSS utility classes.
- ref: Optimized the secondary action button, which is now directly styled with Tailwind classes, removing the need for some Chakra UI components.
- ref: Refactored the Illustration component to remove the dependency on the Chakra UI Icon component.
- ref: The SVG is now rendered directly with native elements, simplifying the implementation and eliminating the need to pass props to the Icon component.

## Preview
Screenshot 2024-10-15 at 14 16 22

## Issues

This closes #197

### CODE REVIEW

**Comment by marco-souza** [comment]
@Joel-leal integration test is failing. Try to run it locally and fix the test before we review it

File: tailwind.config.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import type { Config } from "tailwindcss";
 const config: Config = {
   content: ["./app/**/*.{ts,tsx}", "./packages/**/*.{ts,tsx}"],
   plugins: [daisyuiPlugin],
+  darkMode: "class",
```
Comment (marco-souza) [inline tailwind.config.ts:7]: This should not be part of the current PR, this is related to the #184

File: packages/components/CallToActionSection.tsx [hunk 2]
```diff
@@ -13,28 +13,31 @@ export default function CallToActionSection() {
...
+        </h2>
 
         <p className="text-gray-500 max-w-3xl">{t("description")}</p>
 
-        <Stack spacing={6} w={{ base: "100%", sm: "auto" }} direction={{ base: "column", sm: "row" }}>
-          <Button as="a" href={links.secondaryButton} rounded="full" px={6}>
+        <div className="w-full sm:w-auto flex flex-col sm:flex-row">
+          <a
+            href={links.secondaryButton}
+            className="rounded-full px-6 py-2 font-semibold bg-gray-100 text-black hover:bg-gray-200"
```
Comment (marco-souza) [inline packages/components/CallToActionSection.tsx:32]: Let's use Daisy UI button for adding styles here:
- https://daisyui.com/components/button/

**Review by marco-souza — CHANGES_REQUESTED** [review]
we need to remove the dependencies from `@chakra-ui/react`
- https://github.com/podcodar/webapp/pull/198/files#diff-9535dd9e03701452e6d9fe9dee3acf4395db41c15d38712d2b5e51fb90f6b301R3

**Comment by marco-souza** [comment]
@Joel-leal Changes to the PR description:
- use the keywords `This closes #197`
- use markdown items to create an unordered list for your changes

File: tailwind.config.ts (hunk 1 above)
Comment (Joel-leal) [reply tailwind.config.ts]: change here [7864ad4](https://github.com/podcodar/webapp/pull/198/commits/7864ad4c2075b4cb5fd36107208cdef3773f639b)

**Review by marco-souza — APPROVED** [review]
Thank you 👏🏼

**Review by marco-souza — CHANGES_REQUESTED** [review]
Just forgot, we must use the keywords: `This closes #197` at the PR description, or else, the GitHub ticket automation don't work

**Comment by Joel-leal** [comment]
Change in the description completed.

**Review by marco-souza — APPROVED** [review]

**Pull request merged**